Makefile

```diff
@@ -201,10 +205,15 @@ ifeq (,$(wildcard out/$(BUILDTYPE)/cctest))
 endif
 	@out/$(BUILDTYPE)/cctest --gtest_list_tests
 
+# Rebuilds deps/v8 as a git tree, pulls the third-party dependencies
```

Comment:
Tiny nit and probably a bit onerous, but can we put the directories in this doc in backticks? So `deps/v8` instead of deps/v8? Similarly elsewhere.

Reply:
WHoops! Ignore! I thought I was looking at a markdown file. Sorry.